tests/tools/cdot_geospatial_api_test.py

```diff
@@ -0,0 +1,71 @@
+from datetime import datetime, timedelta, timezone
```

Comment:
Looks like several of these imports aren't used, can we clean up?

Reply:
removed